src/docs/product/performance/filters-display/index.mdx

```diff
@@ -42,7 +42,7 @@ When you select "All Transactions", the table includes all transactions across a
 
 ### Transaction Name Search
 
-On the **Performance** page, the search bar filters by transaction name only, rather than searching for transactions by matching event properties.
+On the **Performance** page, the search bar filters by transaction name only, rather than searching for transactions by matching event properties. You can search for specific transaction names as well as [wildcards](/product/sentry-basics/search/#wildcards).
```

Comment:
```suggestion
Rather than searching for transactions by matching event properties, the search bar on the **Performance** page filters by transaction name only. You can search for specific transaction names or use wildcards, which can take the place of one or more characters in a search term.
```
I agree that it would make more sense to add a bit more of a description of how to incorporate wildcards in search here, rather than linking out to a different section of docs. I'd add "For example....[specific example]." At the end of the paragraph above.